dropshot/src/config.rs

```diff
@@ -49,6 +50,10 @@ pub struct ConfigDropshot {
     pub bind_address: SocketAddr,
     /// maximum allowed size of a request body, defaults to 1024
     pub request_body_max_bytes: usize,
+    /// maximum size of any page of results
```

Comment:
do we want to add `#[non_exhaustive]` to the struct? Would that force consumers to use `..Default::default()` when constructing this? i.e. such that adding more fields would **not** be a breaking change?

Reply:
I wondered the same thing.  Consumers _can_ already do that if they want.  It's kind of nice that the way it is now you can have your code break if we add some new config option so that you get a chance to look yourself at whether our default works for you.  I think I've used this in the past.